.github/workflows/stale.yml

```
@@ -7,21 +7,20 @@ name: Mark stale issues and pull requests
       issues: write
       pull-requests: write
 
     steps:
-    - uses: actions/stale@v5
-      with:
-        repo-token: ${{ secrets.GITHUB_TOKEN }}
-        stale-issue-message: 'Stale issue message'
-        stale-pr-message: 'Stale pull request message'
-        stale-issue-label: 'no-issue-activity'
-        stale-pr-label: 'no-pr-activity'
+      - uses: actions/stale@v5
```

actionlint (1.7.4)

the runner of "actions/stale@v5" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)